beacon-chain/rpc/eth/beacon/sync_committee.go

```diff
@@ -59,6 +57,10 @@ func (bs *Server) ListSyncCommittees(ctx context.Context, req *ethpbv2.StateSync
 				*req.Epoch, currentEpoch,
 			)
 		}
+		if reqPeriodStartEpoch > currentPeriodStartEpoch {
```

Comment:
If someone requests more than 1 epoch into the future, I think it would be better to return an error rather than give them the next epoch's data.